external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/KafkaSpoutEmitTest.java

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
We should have separate PR for 1.x branch as well, or change current implementation to be compatible with JDK 7.